packages/servicenow_cmdb/data_stream/event/manifest.yml

```
@@ -0,0 +1,498 @@
+          #    8G+DZ3TUDYxqMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAG5D
+          #    874A4YI7YUwOVsVAdbWtgp1d0zKcPRR+r2OdSbTAV5/gcS3jgBJ3i1BN34JuDVFw
+          #    3DeJSYT3nxy2Y56lLnxDeF8CUTUtVQx3CuGkRg1ouGAHpO/6OqOhwLLorEmxi7tA
+          #    H2O8mtT0poX5AnOAhzVy7QW0D/k4WaoLyckM5hUa6RtvgvLxOwA0U+VGurCDoctu
+          #    8F4QOgTAWyh8EZIwaKCliFRSynDpv3JTUwtfZkxo6K6nce1RhCWFAsMvDZL8Dgc0
+          #    yvgJ38BRsFOtkRuAGSf6ZUwTO8JJRRIFnpUzXflAnGivK9M13D5GEQMmIl6U9Pvk
+          #    sxSmbIUfc2SGJGCJD4I=
+          #    -----END CERTIFICATE-----
+  - input: aws-s3
+    template_path: aws-s3.yml.hbs
+    title: ServiceNow tables
+    description: Collect table logs from ServiceNow CMDB instance via AWS S3 or SQS.
+    enabled: false
+    vars:
+      - name: collect_s3_logs
+        required: true
+        show_user: true
+        title: Collect logs via S3 Bucket
+        description: To Collect logs via S3 bucket enable the toggle switch. By default, it will collect logs via SQS Queue.
+        type: bool
+        multi: false
+        default: false
+      - name: bucket_arn
+        type: text
+        title: '[S3] Bucket ARN'
+        multi: false
+        required: false
+        show_user: true
+        description: It is a required parameter for collecting logs via the AWS S3 Bucket.
+      - name: queue_url
+        type: text
+        title: '[SQS] Queue URL'
+        multi: false
+        required: false
+        show_user: true
+        description: URL of the AWS SQS queue that messages will be received from. It is a required parameter for collecting logs via the AWS SQS.
+      - name: access_key_id
+        type: password
+        title: Access Key ID
+        multi: false
+        required: false
+        show_user: true
+        description: First part of access key.
+        secret: true
+      - name: secret_access_key
+        type: password
+        title: Secret Access Key
+        multi: false
+        required: false
+        show_user: true
+        description: Second part of access key.
+        secret: true
+      - name: session_token
+        type: password
+        title: Session Token
+        multi: false
+        required: false
+        show_user: true
+        description: Required when using temporary security credentials.
+        secret: true
+      - name: bucket_list_prefix
+        type: text
+        title: "Bucket Prefix"
+        multi: false
+        required: true
+        show_user: true
+        description: Prefix to fetch data from AWS S3 bucket or AWS SQS queue.
+      - name: table_name
+        type: text
+        title: Table Name
+        description: Table name to fetch logs for. Please enter the table name in lowercase with underscores. For example- cmdb_ci_infra_service for the Infrastructure Service table. Refer this [link](https://<your_instance_id>.service-now.com/now/nav/ui/classic/params/target/sys_db_object_list.do) for the exact table names.
```

```suggestion
        description: Table name to fetch logs for. Please enter the table name in lowercase with underscores. For example, cmdb_ci_infra_service for the Infrastructure Service table. Refer this [link](https://<your_instance_id>.service-now.com/now/nav/ui/classic/params/target/sys_db_object_list.do) for the exact table names.
```
Note issue with broken link.